h2o-r/h2o-package/R/gbm.R

```diff
@@ -317,5 +317,8 @@ h2o.gbm <- function(x,
 
   # Error check and build model
   model <- .h2o.modelJob('gbm', parms, h2oRestApiVersion=3, verbose=verbose)
+
+  # Provide actual ntree value 
+  model@model$ntree_actual <- model@model$model_summary['number_of_trees'][,1]
```

Comment:
Should we not provide a similar convenience method as we have in Python? Especially for the parity of APIs.

Reply:
After clarification with @ledell and @Pscheidl this version is considered to be sufficient since we don’t have accessor method for all the other model parameters (just a few) + it will be implemented for all tree-based algorithms.

Reply:
Thanks @koniecsveta